### PR TITLE
[secure-storage] some cleanups

### PR DESCRIPTION
**Pull request by davidiw**
This PR seeks to simplify some of the constructions in secure-storage to be a bit more accessible / less overhead for reviewers.

### CODE REVIEW

File: secure/storage/src/tests/suite.rs [hunk 1]
```diff
@@ -191,8 +191,9 @@ fn test_create_key_pair_and_perform_rotations(storage: &mut dyn Storage) {
         let new_public_key = storage.rotate_key(CRYPTO_NAME).unwrap();
         let new_private_key = storage.export_private_key(CRYPTO_NAME).unwrap();
 
-        let exported_key =
-            storage.export_private_key_for_version(CRYPTO_NAME, public_key).unwrap();
+        let exported_key = storage
```
Comment (JoshLind) [inline secure/storage/src/tests/suite.rs:194]: I'm surprised the linter didn't catch this?

Comment (davidiw) [reply secure/storage/src/tests/suite.rs:194]: what though?

**Comment by davidiw** [comment]
landing this: https://github.com/libra/libra/pull/4031
which contains both PRs

File: secure/storage/src/tests/suite.rs (hunk 1 above)
Comment (JoshLind) [reply secure/storage/src/tests/suite.rs:194]: Ooh sorry -- I was confused about why this line changed wrapping? It seemed to be 2 lines before but then changed to 3 😕

**Pull request closed without merging**